pkg/ottl/parser.go

```diff
@@ -32,6 +32,24 @@ func (e *ErrorMode) UnmarshalText(text []byte) error {
 	}
 }
 
+type LogicOperation string
```

Comment:
I see the logic for having this in this file, but do you think it may make sense to have this and `ErrorMode` in a `config.go` file or similar?

Reply:
Agreed.  I'll do that in a followup PR to try and keep the amount of changes here to a minimal.